### DISCUSSION TITLE
How do I add the latest ChatGPT model (gpt-4o-2024-08-06) to the Model Name drop down?

### DISCUSSION THREAD

**Discussion by taylorbayouth** [langflow-ai/langflow#3777] [General]
Title basically is my question. I'm using the "OpenAI" node and there is a dropdown with the model names. I want to test the latest from OpenAI so wonder how/where I can do that. Do I need to create a custom component? Or is there a definitions file somewhere I can edit?

**Comment by jjmata** [comment]
If you are running from source, you should be able to just add what you need here:
* https://github.com/langflow-ai/langflow/blob/79a1257b7dfad21cb420cf166cfd65d89d21c1bf/src/backend/base/langflow/base/models/openai_constants.py#L4
